safe_transaction_service/safe_messages/tests/test_views.py

```diff
@@ -269,13 +269,13 @@ def test_safe_message_add_signature_view(self, get_owners_mock: MagicMock):
             data=data,
         )
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(
+        self.assertDictEqual(
             response.data,
             {
-                "non_field_errors": [
+                "signature": [
                     ErrorDetail(
-                        string="1 owner signature was expected, 0 received",
-                        code="invalid",
+                        string="Ensure this field has at least 1 hexadecimal chars (not counting 0x).",
```

Comment:
Why the error message changed?

Reply:
Because previously we were not requiring minimum length for the signature in the validator, and now we can discard the signature without further validation